### PR TITLE
Add UI tests for pixelation level saving [finishes #102962296]

### PR DESCRIPTION
**Pull request by davidsbailey**
finishes https://www.pivotaltracker.com/story/show/102962296

### CODE REVIEW

File: dashboard/test/ui/features/pixelation.feature [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    And pixelation data has text "0000 0100 0000 0010 0000 0011 000 111 100 010 001 110"
+
+    When I press keys "111" for element "#pixel_data"
+    And pixelation data has text "0000 0100 0000 0010 0000 0011 000 111 100 010 001 110 111"
+    And I finish pixelation level and reload
+    Then pixelation data has text "0000 0100 0000 0010 0000 0011 000 111 100 010 001 110 111"
+
+    When I click selector "input[value='hex']"
+    And pixelation data has text "04 02 03 1E23B8"
+    And I press keys "e1" for element "#pixel_data
```
Comment (Bjvanminnen) [inline dashboard/test/ui/features/pixelation.feature]: missing close quote at end of line

**Comment by Bjvanminnen** [comment]
lgtm

**Comment by Bjvanminnen** [comment]
I would have thought that the tests complained about the missing quote when run. Was that not the case?

**Comment by davidsbailey** [comment]
I ran the tests, and then did "one last thing" (add a comment somewhere unrelated) before I pushed them. I must have somehow removed the quote when I added the comment. Good catch.

**Comment by Bjvanminnen** [comment]
I only noticed because github does syntax highlighting :)

**Pull request merged**